### ISSUE TITLE
[extension-browser] UX: Colors are not accessible in FF dark mode

### ISSUE THREAD

**Issue by ststimac** [webhintio/hint#1504]
Running Firefox Developer tools in dark mode: 

![webhint-dark-theme-ff](https://user-images.githubusercontent.com/18073131/48518891-e0538400-e81f-11e8-94f9-ce7d0ff57250.PNG)

The code results are not accessible due to color contrast. A short term solution would just make the code container box white, but that's also incredibly harsh contrast-wise. 

I will work on getting some color codes for the intial dark theme for @antross

**Comment by molant** [comment]
We are using [highlightjs](https://highlightjs.org/) so maybe there's a theme that can be used for dark mode.
